src/topic_training/src/Demo.py

```diff
@@ -0,0 +1,92 @@
+''' Import rospy to use ROS with python. '''
+import rospy
+''' Import the message type we will be publishing. '''
+from std_msgs.msg import String
+
+''' Constant variables to be used below. '''
+QUEUE_SIZE = 10
+RATE = 10
+
+'''
+This function will get called when a new message has 
+arrived on the topic.
+'''
+def callback(data):
+    ''' Print out the message to the terminal. '''
+    rospy.loginfo('I heard: [%s]', data.data)
+
+''' 
+The main method that contains the creation of the node, 
+Publisher, Subcriber, and topic. It also describes what 
+we will do with the data/message. 
+'''
+def main():
+    ''' Initialize ROS and specify the name of the node 
+        (which is the name of the file in this case). '''
+    rospy.init_node('Demo', anonymous=True)
+    
+    ''' Initialize the Publisher. State the name of the 
+        topic ('chatter'), the message type we will be 
+        publishing (String, which is actually in the 
+        class std_msgs.msg.String), and the queue size 
+        (1000). The queue size is the maximum number of 
+        messages kept as a buffer before throwing away 
+        old ones if we are publishing too quickly. '''
+    pub = rospy.Publisher('chatter', String, queue_size=QUEUE_SIZE)
```

Comment:
I would keep some `std_msgs` namespace here to signify that the message type is coming from there.

Reply:
@rusmilov Is this comment addressed?

Reply:
Fixed in 211a686.